src/components/Text.js

```diff
@@ -22,20 +22,17 @@ export const RegularText = styled.Text`
 
 export const BoldText = styled(RegularText)`
   font-family: titillium-web-bold;
-`;
-
-export const LightestText = styled(RegularText)`
-  color: ${colors.lightestText};
 
   ${(props) =>
-    props.bold &&
+    props.light &&
```

Comment:
Why do you have a `light` prop now for `BoldText`? Cannot read about it in the commit message.
It somehow makes no sense for me, because "light" text for bold is equal to `RegularText`, isn't it?

Reply:
it's not a good prop name , it's used on PriceCard for prices  a small(normalize(14)) bold and white  text .  I can maybe call it white ?

Reply:
What about `<LightestText small...` for that cases?

Reply:
TODO: restructure RegularText will be jest Text with 4 different props : bold , small, link and white

Reply:
consider `lightest` instead of `white`. and we can change `link` to `primary` also, which would be more general.